### PR TITLE
feat: add valueChangeOptions as an optional metadata parameter

### PR DESCRIPTION
**Pull request by raman325**
As discussed here: https://github.com/zwave-js/node-zwave-js/issues/1321#issuecomment-760086568

This would make it easier for us to build logic to determine when we can use the `transitionDuration` option

### CODE REVIEW

**Comment by raman325** [comment]
@zwave-js-bot fix lint

**Comment by raman325** [comment]
@zwave-js-bot fix lint

**Comment by raman325** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
@raman325 We're going to do this, but this needs some (linguistic) revision.

I want to strongly-type the array options, for that reason the interface needs to be moved into `@zwave-js/core`. We cannot call it `SetValueOptions` there, because that interface already exists for the `ValueDB.setValue` call options. I'm hesitant to call it its current name `SetValueAPIOptions` there, because that API is a concept of the `zwave-js` module, not the `core` module.

I'm open to suggestions, maybe `RelatedValues` or something with `Meta` in the name (although that can be confusing with metadata)? Ideally, the name should make sense for the Metadata property (currently called `setValueOptions`) aswell.

**Comment by AlCalzone** [comment]
I think `valueChangeOptions` could work. Thoughts?

**Comment by raman325** [comment]
Works for me!

**Review by AlCalzone — APPROVED** [review]

File: docs/api/valueid.md [hunk 1]
```diff
@@ -60,6 +61,7 @@ interface ValueMetadataAny {
 -   `description`: A description of the value
 -   `label`: A human-readable label for the property
 -   `ccSpecific`: CC specific information to help identify this value [(see below)](#CC-specific-fields)
+-   `setValueOptions`: Parameters that can be passed as `options` to the [`setValue`](api/node.md#setvalue) command for this value.
```
Comment (MartinHjelmare) [inline docs/api/valueid.md:64]: Is this name correct, `setValueOptions`?

Comment (raman325) [reply docs/api/valueid.md:64]: good catch, way to be on vacation. Will submit a new PR

Comment (AlCalzone) [reply docs/api/valueid.md:64]: Whoops I forgot to change that when I renamed things

**Pull request merged**